system/iptables_raw.py

```diff
@@ -304,6 +321,12 @@ def _check_compatibility(self):
         else:
             Iptables.module.fail_json(msg="Could not fetch iptables version! Is iptables installed?")
 
+    # Check if /etc/init.d/iptables-persistent for older Debian based systems
+    # Check if /etc/init.d/netfilter-persistent for newer Debian based systems
+        if self.is_debian():
+            if not os.path.isfile('/etc/init.d/netfilter-persistent') and not os.path.isfile('/etc/init.d/iptables-persistent'):
+                Iptables.module.fail_json(msg="Could not detect iptables-persistent package. Is iptables-persistent installed?")
```

Comment:
I talked with a few Debian/Ubuntu users and as far as I understand not everyone uses `iptables-persistent`, which is why I don't want to make the package mandatory. I agree that on Debian save paths should be `/etc/iptables/rules.v[46]`, but the package doesn't need to be installed.